src/log.cpp

```diff
@@ -77,7 +77,7 @@ const char* ChannelToString(LogChannel channel) {
   }
 }
 
-void ConsoleWrite(const char* str) { printf("%s", str); }
+void ConsoleWrite(const char* str) { fprintf(stderr, "%s", str); }
```

Comment:
Hm, not sure using stderr for general logs makes sense. Is there any particular issue with buffering?

Reply:
If it crashes on like a lua load for example, logs won't get flushed out and you'll miss the log for the error. We could flush after every print as well.

Reply:
Let's revert that change and send me an example of the issue in Discord, I'll try to take a look at it